src/PostgREST/QueryBuilder.hs

```diff
@@ -418,7 +418,7 @@ pgFmtFilter table (Filter fld (Operation hasNot_ ex)) = notOp <> " " <> case ex
    (op, VText val) -> pgFmtFieldOp op <> " " <> case op of
      "like"  -> unknownLiteral (T.map star val)
      "ilike" -> unknownLiteral (T.map star val)
-     "@@"    -> "to_tsquery(" <> unknownLiteral val <> ") "
```

Comment:
That's what I've done. I left the original test, copied it, and in the copy changed the operator. I've just checked again and can't spot what error I've made.